lib/models/updateAction.js

```diff
@@ -135,7 +150,7 @@ function doRequest(action, event, token, callback) {
                 attributes: options.attributes
             }
         ],
-        updateAction: 'APPEND'
+        updateAction: 'append'
```

Comment:
Thid doRequest() is used by the NGSIv1 logic?

* In that case APPEND is correct
* If only used by NGSIv2 logic, then "append" is correct

Reply:
fixed in a01067872a33567608238ec7683be42e35565137